tests/api/api_routes/builds/test_build_app.py

```diff
@@ -141,6 +141,29 @@ def test_build_app_handle_errors(
             "[/api/builds/:app] Error while tagging and building the app",
         )
 
+    def test_build_app_handle_early_errors(
```

Comment:
Same as https://github.com/ChauffeurPrive/nestor-api/pull/41/files#r455779902, new `coverage` lib needs to has `if` called with true and false